.circleci/config.yml

```diff
@@ -1,73 +1,121 @@
+references:
+  #
+  # Environment
+  #
+  shared-environment: &shared-environment
+    CIRCLE_ARTIFACTS: /tmp/artifacts
+    CIRCLE_TEST_REPORTS: /tmp/test_results
+
+  #
+  # Image
+  #
+  docker-image: &docker-image
+    - image: circleci/node:10.10.0-browsers
+
+  #
+  # Artifact and test results
+  #
+  setup-results-and-artifacts: &setup-results-and-artifacts
+    name: Create Directories for Results and Artifacts
+    command: |
+      mkdir -p                             \
+        "$CIRCLE_ARTIFACTS/translate"      \
+        "$CIRCLE_TEST_REPORTS/client"      \
+        "$CIRCLE_TEST_REPORTS/integration" \
+        "$CIRCLE_TEST_REPORTS/server"
```

Comment:
CI will "categorize" test results. Each of these may have their own summary box if there's a report.